lightning/src/onion_message/packet.rs

```diff
@@ -314,6 +314,21 @@ pub(crate) enum ControlTlvs {
 	Receive(ReceiveTlvs),
 }
 
+impl ControlTlvs {
+	pub(crate) fn pad_tlvs(mut self, max_length: usize) -> Self {
```

Comment:
Let's call this `pad_to_length` given "tlvs" is already part of the type name.

Reply:
Done! [pr3177.10](https://github.com/shaavan/rust-lightning/commits/pr3177.10)
Thanks for the pointer!